docs/installation.rst

```diff
@@ -101,7 +103,33 @@ and insert the following contents towards the end of the file
         Require all granted
     </Directory>
 
-Restart Apache:
+Note: be sure to keep the number of threads limited to 1 in the WWSGIDaemonProcess
```

Comment:
I think we could put this in a ``..warning`` block to really emphasise it --- this is a nasty gotcha!
Also, extra 'W' in WWSGIDaemonProcess